src/clj/rems/api/applications.clj

```diff
@@ -171,7 +171,7 @@
 
     (GET "/export" []
       :summary "Export all submitted applications of a given form as CSV"
-      :roles #{:owner}
+      :roles #{:owner :organization-owner}
```

Comment:
hmm I wonder if this should be available for reporter also

Reply:
Removed organization-owner from here now, as the same considerations apply as with the other API endpoints. We can add both organization-owner and reporter here later.